src/Cake.Common.Tests/Fixtures/Tools/OctopusDeployPusherFixture.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.Collections.Generic;
+using Cake.Common.Tools.OctopusDeploy;
+using Cake.Core.IO;
+using Cake.Testing.Fixtures;
+
+namespace Cake.Common.Tests.Fixtures.Tools
+{
+    internal sealed class OctopusDeployPusherFixture : ToolFixture<PushPackageSettings>
+    {
+        public List<FilePath> Packages { get; set; }
+        public OctopusDeployPusherFixture() : base("Octo.exe")
+        {
+            Packages = new List<FilePath>
+            {
+                "MyPackage.1.0.0.zip",
+                "MyOtherPackage.1.0.1.nupkg"
+            };
```

Comment:
@agc93 I was referring to this:

![image](https://cloud.githubusercontent.com/assets/1271146/16149780/e619ae64-348a-11e6-8700-115dc713f384.png)

I don't think we have a fixed rule for this particular situation.